core/strings/strings.odin

```diff
@@ -1891,6 +1891,48 @@ index_multi :: proc(s: string, substrs: []string) -> (idx: int, width: int) {
 	return
 }
 /*
+Finds the last occurrence of any substring in `substrs` within `s`
+
+Inputs:
+- s: The string to search in
+- substrs: The substrings to look for
+
+Returns:
+- idx: the index of the last matching substring
+- width: the length of the found substring
+*/
+last_index_multi :: proc(s: string, substrs: []string) -> (idx: int, width: int) {
+	idx = -1
+	if s == "" || len(substrs) <= 0 {
+		return
+	}
+	// disallow "" substr
+	for substr in substrs {
+		if len(substr) == 0 {
+			return
+		}
+	}
+
+	highest_index := -1
+	found := false
+	for substr in substrs {
+        haystack_slice_start := max( 0 , highest_index )
```

Comment:
This line is indented differently from the others, also has some weird formatting where you added a space after and before the parentheses and comma which we don't do.